gbpservice/nfp/orchestrator/db/nfp_db.py

```diff
@@ -202,7 +202,7 @@ def _set_mgmt_port_for_nfd(self, session, network_function_device_db,
                 session.add(port_info_db)
             session.flush()
             nfd_db.mgmt_port_id = port_info_db['id']
-            del network_function_device['mgmt_port_id']
+            # del network_function_device['mgmt_port_id']
```

Comment:
not sure though, can you check all the other functions and help in modifying, nothing is getting effected though.